### ISSUE TITLE
Incident Profile page scrolls down to middle of page

### ISSUE THREAD

**Issue by flavour** [sahana/WACOP#30]
e.g. http://cad.aidiq.com/eden/event/incident/23/custom
This same behaviour doesn't happen with the Event profile page even though they're very similar:
e.g. http://cad.aidiq.com/eden/event/event/2/custom

I have dug around a lot to try & ascertain what may be causing this but have come up blank.
If I comment the whole incident-resources DIV then the problem goes away but it is present no matter which elements inside the div are present. I've hunted around the JS for a focus() and looked to see if there's something in the CSS that could do this, but am unable to find anything.

Perhaps this is something that you can identify easily?

**Comment by dhornbein** [comment]
I'm seeing the same behavior, one thing I've noticed is that it doesn't happen on the first page load.

Upon further investigation I find the behavior inconsistent. I navigated to `/eden/event/incident/23/custom#` and didn't see the jump, then refreshed the page and saw the jump.

Opened the same page in another tab (same session) and didn't see the jump, refreshed and still no jump.

What I found is that the page only scrolls down to the middle when the Google Chrome Inspector is open. It still feel a bit inconsistent, but that might be an avenue of investigation.
